packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +123,41 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      headers: Headers
```

Comment:
to maintain the same structure (even if the logger in these functions is not used) could the last parameter be a `context: WithLogger<BffAppContext>` instead of a simple `Header`?